### ISSUE TITLE
Allow system_site_packages=True in Virtualenv.create()

### ISSUE THREAD

**Issue by PhoenixHe-NV** [pex-tool/pex#1973]
Currently PEX could create virtualenv to populate bundled packages, and in PEX code it allows to [create venv with "--system-site-packages"](https://github.com/pantsbuild/pex/blob/22e651589b3f2da26da6786dcbd1553fd7c8d4a6/pex/venv/virtualenv.py#L141), but there is no way to specify this parameter at runtime.

We want to run with venv and system-site-packages enabled, can we have an extra environment variable to specify this behavior?

**Comment by jsirois** [comment]
Did you know you can do this?:
```
PEX_TOOLS=1 ./my.pex venv here/ --help
```

Or use the `pex-tools` console script available when you install the `pex` tool:
```
pex-tools ./my.pex --help
pex-tools ./my.pex venv --help
```

Does that satisfy your requirements?

**Comment by PhoenixHe-TSP** [comment]
@jsirois Thanks for your advice! It's a good idea to specify the option in pex-tools, but seems like it doesn't pass system-site-packages

https://github.com/pantsbuild/pex/blob/52c84031bc22781f59e6acba74be9c5736fb3598/pex/tools/commands/venv.py#L177-L183

**Comment by jsirois** [comment]
Aha, right. So, if a `pex-tools venv --system-site-packages` option were added, would that work for you then? If so I'll mark this as a feature request and if you want to implement that feature I'm happy to review; otherwise it will go into the current backlog tracked in #1975.

**Comment by PhoenixHe-TSP** [comment]
@jsirois Adding a new option `--system-site-packages` would work for me. I don't have enough time to implement this feature, but I could modify pyenv.cfg after venv extracted. Anyway, please add this to backlog, thanks!

**Comment by PhoenixHe-TSP** [comment]
@jsirois Now I'm blocked without this feature. If I manually modify `pyvenv.cfg` to change to `include-system-site-packages = true`, but the “pip” is extracted without `--system-site-packages`, then if I packed "pip", in extracted venv's pip, it will have `#!/tmp/pex_venv/bin/python3.8 -sE` at the first line, which will let `pip list` skip user local site ($HOME/.local)

Maybe I'll try to make a PR on this issue...

**Comment by jsirois** [comment]
Alright @PhoenixHe-NV, `--system-site-packages` support is now available in Pex 2.16.0.
